### PR TITLE
[Icon] Update clipboard icon to look like a clipboard

### PR DESCRIPTION
**Pull request by zecarlostorre**
### WHY are these changes introduced?

Current icon doesn't look like a clipboard, and sometimes is confused for a trashcan.

### WHAT is this pull request doing?

Updating the icon:

image

### CODE REVIEW

File: .changeset/wicked-moose-relax.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris-icons': patch
+---
+
+Update clipboard icon
```
Comment (kyledurand) [inline .changeset/wicked-moose-relax.md]:
```suggestion
Updated clipboard icon
```

File: polaris-icons/icons/ClipboardMinor.yml [hunk 2]
```diff
@@ -4,9 +4,10 @@ description: Used to represent a clipboard, usually accompanying a copy action.
 keywords:
   - clipboard
   - copy
+  - copy and paste
 authors:
-  - Adam Whitcroft
-version: 1
+  - José Torre
```
Comment (kyledurand) [inline polaris-icons/icons/ClipboardMinor.yml:10]:
```suggestion
  - Adam Whitcroft
  - José Torre
```

**Review by kyledurand — APPROVED** [review]

File: polaris-icons/icons/ClipboardMinor.svg [hunk 3]
```diff
@@ -1 +1 @@
-<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="M15 2a1 1 0 011 1v13.5a1.5 1.5 0 01-1.5 1.5h-9A1.5 1.5 0 014 16.5V3a1 1 0 112 0v1a2 2 0 002 2h4a2 2 0 002-2V3a1 1 0 011-1zm-4 2H9a1 1 0 110-2h2a1 1 0 110 2z" fill="#5C5F62"/></svg>
\ No newline at end of file
+<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="m8 3c0-0.55228 0.44772-1 1-1h2c0.5523 0 1 0.44772 1 1 0 0 0 0.99713 0.9957 1h2.0043c1.1046 0 2 0.89543 2 2v7.5858c0 0.5304-0.2107 1.0391-0.5858 1.4142l-2.4142 2.4142c-0.3751 0.3751-0.8838 0.5858-1.4142 0.5858h-7.5858c-1.1046 0-2-0.8954-2-2v-10c0-1.1046 0.89543-2 2-2h2.0043c0.99569-0.00287 0.99569-1 0.99569-1zm5 3v1c0 0.55228-0.4477 1-1 1h-4c-0.55228 0-1-0.44772-1-1v-1h-2v10h7v-2c0-0.5523 0.4477-1 1-1h2v-7h-2z" clip-rule="evenodd" fill="#5C5F62" fill-rule="evenodd"/></svg>
```
Comment (kyledurand) [inline polaris-icons/icons/ClipboardMinor.svg]:
```suggestion
<svg viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path d="m8 3c0-0.55228 0.44772-1 1-1h2c0.5523 0 1 0.44772 1 1 0 0 0 0.99713 0.9957 1h2.0043c1.1046 0 2 0.89543 2 2v7.5858c0 0.5304-0.2107 1.0391-0.5858 1.4142l-2.4142 2.4142c-0.3751 0.3751-0.8838 0.5858-1.4142 0.5858h-7.5858c-1.1046 0-2-0.8954-2-2v-10c0-1.1046 0.89543-2 2-2h2.0043c0.99569-0.00287 0.99569-1 0.99569-1zm5 3v1c0 0.55228-0.4477 1-1 1h-4c-0.55228 0-1-0.44772-1-1v-1h-2v10h7v-2c0-0.5523 0.4477-1 1-1h2v-7h-2z" fill="#5C5F62" /></svg>
```
I think you can get rid of both of these attributes.

**Review by sarahill — APPROVED** [review]
🎉

File: polaris-icons/icons/ClipboardMinor.yml (hunk 2 above)
Comment (zecarlostorre) [reply polaris-icons/icons/ClipboardMinor.yml:10]: Ah, my bad. I didn't think I should keep the name of the designer of the previous one.

**Pull request merged**